salt/beacons/napalm_beacon.py

```diff
@@ -286,8 +286,8 @@ def validate(config):
     if not isinstance(config, list):
         return False, 'Configuration for napalm beacon must be a list.'
     for mod in config:
-        fun = mod.keys()[0]
-        fun_cfg = mod.values()[0]
+        fun = list(mod.keys())[0]
```

Comment:
This can just be `list(mod)[0]` - the default creating a list from mapping is to use the keys.